src/components/views/elements/SettingsFlag.tsx

```diff
@@ -103,6 +104,7 @@ export default class SettingsFlag extends React.Component<IProps, IState> {
                         onChange={this.onChange}
                         disabled={this.props.disabled || !canChange}
                         aria-label={label}
+                        tooltip={this.props.disabled && this.props.disabledTooltip}
```

Comment:
this is the line that makes it feel like `disabledTooltip` should be a boolean, so let's avoid the magic syntax in this case:
```suggestion
                        tooltip={this.props.disabled ? this.props.disabledTooltip : null}
```